### ISSUE TITLE
The "rewindAndfastForward" buttons do not respond on iOS and Android.

### ISSUE THREAD

**Issue by bestriser** [zezo357/flutter_meedu_videoplayer#63]
The "rewindAndfastForward" buttons do not respond on iOS and Android.

When I press the "rewindAndfastForward" button, the console displays the message "flutter: set tapped twice to false".

Is there anything I can do?

### pubspec.yaml
```yaml
  flutter_meedu_videoplayer: ^4.2.3
  media_kit_libs_ios_video: ^1.0.4

  ## TODO(yamamura): investigate the error when androidUseMediaKit is set to true and the video playback screen is displayed.
  ## media_kit_libs_android_video: ^1.0.3
```

### main.dart
```dart
void main() async {
  WidgetsFlutterBinding.ensureInitialized();
  
  // TODO(yamamura): investigate the error when androidUseMediaKit is set to true and the video playback screen is displayed.
  await initMeeduPlayer(iosUseMediaKit: true, /* androidUseMediaKit: true */);
  );
}
```

### Flutter Doctor
```
[✓] Flutter (Channel stable, 3.7.12, on macOS 13.3.1 22E261 darwin-arm64, locale ja-JP)
    • Flutter version 3.7.12 on channel stable at /Users/yuyayamamura/fvm/versions/stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4d9e56e694 (2 weeks ago), 2023-04-17 21:47:46 -0400
    • Engine revision 1a65d409c7
    • Dart version 2.19.6
    • DevTools version 2.20.1

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at /Users/yuyayamamura/Library/Android/sdk
    • Platform android-33, build-tools 33.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14E222b
    • CocoaPods version 1.12.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.77.3)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.62.0

[✓] Connected device (4 available)
    • SH RM12 (mobile)            • 357983102288699           • android-arm64  • Android 11 (API 30)
    • bestriserのiPhoneXs (mobile) • 00008020-000325200A88002E • ios            • iOS 16.4.1 20E252
    • macOS (desktop)             • macos                     • darwin-arm64   • macOS 13.3.1 22E261 darwin-arm64
    • Chrome (web)                • chrome                    • web-javascript • Google Chrome 112.0.5615.137

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Comment by zezo357** [comment]
Test on `flutter_meedu_videoplayer: 4.2.2`
And let me know

**Comment by zezo357** [comment]
I just tested and android is working correctly, can you provide a video on how it fails in your end?

**Comment by bestriser** [comment]
@zezo357 
Thank you for testing this!
I am sending you screen captures of the "rewind" and "fastForward" buttons on iOS and Android.
Only the "rewind" button responds only when in full screen.
These bugs were the same when I turned off MidiaKit.

### iOS
https://user-images.githubusercontent.com/55462291/235853372-ad2b8494-8202-4220-ab5e-f88bb04f8578.mp4

### Android
https://user-images.githubusercontent.com/55462291/235853335-870aad28-a185-402d-992a-c897fe7f9683.mp4

**Comment by zezo357** [comment]
Thank you for videos, will test again and let you know

**Comment by zezo357** [comment]
@bestriser can you provide an example code?

**Comment by bestriser** [comment]
@zezo357 
Thank you for testing.
I built the EXAMPLE and found the same problem.

Could you please check the following file?
`flutter_meedu_videoplayer/package/example/lib/pages/custom_icons_example.dart`

**Comment by zezo357** [comment]
> @zezo357 
> Thank you for testing.
> I built the EXAMPLE and found the same problem.
> 
> Could you please check the following file?
> `flutter_meedu_videoplayer/package/example/lib/pages/custom_icons_example.dart`


Oh, okay will check it and let you know, I will work on it after finishing some work.

**Comment by zezo357** [comment]
@bestriser can you test on github?
pubspec
```
  flutter_meedu_videoplayer:
    git: https://github.com/zezo357/flutter_meedu_videoplayer
```

**Comment by bestriser** [comment]
@zezo357 
thanks!
I tested it after changing `pubspec.yaml` as follows.
Then, rewindAndfastForward button worked on iOS and Android!!
```yaml
  flutter_meedu_videoplayer:
    git:
      url: https://github.com/zezo357/flutter_meedu_videoplayer
      path: package
```

**Comment by zezo357** [comment]
fixed in `4.2.6`
